### ISSUE TITLE
Can't activate all rules starting from second page

### ISSUE THREAD

**Issue by oriontvv** [flask-dashboard/Flask-MonitoringDashboard#120]
It seems that I catched a bug with activation of any rules that paginated not on the first page(after 25)
First 25 rules work well, but for next rule entries ajax request do not sent.
I have Flask-MonitoringDashboard==1.12.5 from pypi and Flask==0.12

**Comment by FlyingBird95** [comment]
Hi oriontvv,

thanks for noticing this bug. I was able to recreate this bug, and it is on this branch now: https://github.com/flask-dashboard/Flask-MonitoringDashboard/tree/issue120.

The problem seems to be that the JQuery script is not executed on the second page: https://github.com/flask-dashboard/Flask-MonitoringDashboard/blob/c90f14295530da8ea8f4d2b473b2d6132f60b991/flask_monitoringdashboard/templates/fmd_rules.html#L46

We'll try to solve this bug, and let you know when it is solved.

P.s. A way to avoid this right now, is to show more entries on the first page ;)

**Comment by FlyingBird95** [comment]
Hi oriontvv,

I've found the source of where the bug was coming from. The data on the rules-page is sent as soon as you click a checkbox. It then sends the data from all checkboxes.

However, as the checkboxes from other pages where not visible, it was somehow never send to the Flask application.

The code for sending the checkboxes has now been updated (as well as the code for processing the data of the checkboxes). The major difference is that a checkbox is now processed individually, instead of all at once.

A PR #121 has been made, and will be merged in development soon. It will take approximately a few days to go from development to master. To conclude, from version 1.12.6 it will be fixed ;)

Thanks again for noticing this bug to us!
